C7Engine/AI/BarbarianAI.cs

```diff
@@ -11,40 +12,89 @@ public class BarbarianAI {
 
 		private ILogger log = Log.ForContext<BarbarianAI>();
 
+		// Threshold at which barbarians take risky actions.
+		// In the future this may vary based on barbarian settings, and perhaps the situation
+		const double COLLEGE_TRY_THRESHOLD = 1.0 / 3;
```

Comment:
Yeah, it's a reference to this idiom in English: https://www.merriam-webster.com/dictionary/give%20it%20the%20old%20college%20try

Chosen because the barbarians will (currently) make an attempt at winning even if the odds aren't great, which is how I've often seen that idiom used.  Going to give something a try but not necessarily expecting a lot of success.  Maybe it's a bit too local of an idiom?  I have lived my entire life in either colleges or college towns...